build/buildBuildImages.sh

```diff
@@ -310,11 +307,15 @@ function buildLtsVersionsImage() {
 }
 
 function buildLatestImages() {
-	buildLtsVersionsImage
+	local debianFlavor=$1
+	if [ -z "$debianFlavor" ] ; then
+		debianFlavor="stretch"
+	fi
+	buildLtsVersionsImage $debianFlavor
 
 	echo
 	echo "-------------Creating latest build images-------------------"
-	local builtImageName="$ACR_BUILD_IMAGES_REPO"
+	local builtImageName="$ACR_BUILD_IMAGES_REPO:debian-$debianFlavor"
```

Comment:
I'm wondering if we need to think about how to handle the `latest` image since it's currently always built using `stretch`, and this would tag the image as `debian-stretch` instead of `latest` or `latest-debian-stretch`, correct?

Reply:
Yes thats right. I was wondering about that particular case as well. Even though we call it the latest image, I think our partners were just using `build:<releasetag>` (https://mcr.microsoft.com/v2/oryx/build/tags/list). `latest` was never a part of the image tag, so I decided not to include it here. I could really go either way on this naming schema though

Reply:
I could go either way as well as long as it's communicated to partners that this specific tag is changing "drastically" relative to other tags that are just getting the OS added in the tag.

Reply:
Ok, take a peek at this commit: https://github.com/microsoft/Oryx/pull/1529/commits/721b6d43c92256978468dd96797512d4cdd6218b

I've added some manual tagging of the latest image with no tag, which adds `build`, `build:latest`, and `build:<releaseTag>` methods of pulling the latest image. I think that this was a good point since this allows us to intentionally set exactly what the "latest" tag means and it stays up to date. We also don't have to update the various places in our docs where we have tutorials for people having them pull the "latest" image.

Can definitely revert the commit if we decide that going with the original strategy is better

Reply:
This approach makes sense to me 👍